examples/react-native/ios/ReactNative/Info.plist

```diff
@@ -1,64 +1,68 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
+<dict>
```

Comment:
Curious abt the changes in this file, are they autogenerated by RN?

Reply:
XCode. I think it's mostly just reordering , what was actually added is 

```
<array>
		<dict>
			<key>CFBundleTypeRole</key>
			<string>Editor</string>
			<key>CFBundleURLName</key>
			<string>myapp</string>
			<key>CFBundleURLSchemes</key>
			<array>
				<string>myapp</string>
			</array>
		</dict>
	</array>
```

For deeplinking

Reply:
Thanks! Bit confused on what the meaning of "add deep linking" is in this PR since the example app already has deep linking support?

Reply:
I guess complete deep linking? Has some updates and aws-exports didn't point to the correct link so wasn't working.